go_modules/lib/dependabot/go_modules/file_updater.rb

```diff
@@ -86,7 +86,7 @@ def file_updater
             credentials: credentials,
             repo_contents_path: repo_contents_path,
             directory: directory,
-            tidy: !@repo_contents_stub
+            tidy: options.fetch(:tidy, false)
```

Comment:
Concern: this should still respect `@repo_contents_stub`.

```suggestion
            tidy: options.fetch(:tidy, !@repo_contents_stub)
```

In the rejected rev from Friday this just set `#tidy` after the guard in the constructor, I think this is the equivalent?

Reply:
Ah, good catch, thanks!